docs/pages/project/compatibility_matrix/compatibility.md

```diff
@@ -0,0 +1,198 @@
+---
+layout: default
+title: Compatibility Matrix
+permalink: project/compatibility-matrix
+description: a complete compatibility matrix and project test status dashboard.
+language: en
+display-title: "false"
+list: exclude
+type: "project"
+---
+
+# Meshery Compatibility Matrix
+
+## Integration Tests
+
+As a key aspect of Meshery, its integrations with other systems are routinely tested. Unit, integration testing occurs before and after every pull request (before code is to be merged into the project and after code is merged into the project). Regression tests are run nightly.
+
+<script type="text/javascript">
```

Comment:
I see that the javascript has been commented out, how is the toggle still working though? 🤔

Reply:
Since it is inside the script tag the html comment will not work.

Reply:
So could you just remove the brackets then we can merge. We'll delete the old results later.

Reply:
Yes ok